### PR TITLE
add simplistic operator-sdk api to test integration

### PR DESCRIPTION
**Pull request by spolti**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

**Comment by spolti** [comment]
CI issue will be fixed by https://github.com/serverlessworkflow/sdk-go/issues/144

File: kubernetes/groupversion_info.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022.
```
Comment (ricardozanini) [inline kubernetes/groupversion_info.go:2]: Do you mind fixing this license header? Maybe we can use the `addheaders` script from other projects.

File: kubernetes/k8s_workflow_integration.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+// This package provides a very simple api for kubernetes operator to test the integration
+// of the Serverless SDK-Go with operator-sdk controller-gen and deepcopy-gen tools.
+// The purpose of this integration is to stop issues like below beforehand:
+// github.com/serverlessworkflow/sdk-go/model/event.go:51:2: encountered struct field "" without JSON tag in type "Event"
+// github.com/serverlessworkflow/sdk-go/model/states.go:66:12: unsupported AST kind *ast.InterfaceType
+
+// SDKServerlessWorkflowSpec defines a base API for integration test with operator-sdk
+type SDKServerlessWorkflowSpec struct {
```
Comment (ricardozanini) [inline kubernetes/k8s_workflow_integration.go:30]:
```suggestion
type ServerlessWorkflowSpec struct {
```
Fits better, don't you think? 😎

File: kubernetes/k8s_workflow_integration.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	BaseWFlow model.BaseWorkflow `json:"inline"`
```
Comment (ricardozanini) [inline kubernetes/k8s_workflow_integration.go:31]:
```suggestion
	BaseWorkflow model.BaseWorkflow `json:"inline"`
```

**Review by ricardozanini — COMMENTED** [review]
Looks neat! Just a couple of minor things.

File: kubernetes/groupversion_info.go (hunk 1 above)
Comment (spolti) [reply kubernetes/groupversion_info.go:2]: sure, fixed.

File: kubernetes/k8s_workflow_integration.go (hunk 2 above)
Comment (spolti) [reply kubernetes/k8s_workflow_integration.go:30]: sounds good.

**Comment by ricardozanini** [comment]
@spolti can you fix the WhiteSource check? The other one, I fix it in #149

**Comment by ricardozanini** [comment]
@spolti can you rebase?

**Review by ricardozanini — APPROVED** [review]
Minor things only, the rest looks pretty good. Maybe if you have the time, add a small CR and CRD yaml example.

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**